src/ast/nodes/ConditionalExpression.ts

```diff
@@ -21,28 +26,45 @@ export default class ConditionalExpression extends NodeBase {
 		path: ObjectPath,
 		callOptions: CallOptions,
 		callback: ForEachReturnExpressionCallback,
-		options: ExecutionPathOptions
+		calledPathTracker: EntityPathTracker
```

Comment:
Again, it is probably obvious that the tracker is for this "called path", so perhaps just `tracker` or `recursionTracker`?

Reply:
👍